.github/workflows/ci.yml

```diff
@@ -40,17 +41,47 @@ jobs:
         run: |
           apk add \
             unzip \
+      - name: Cache mongodb PHP extension build
+        if: (!startsWith(matrix.php, '7.1'))
+        uses: actions/cache@v1
+        with:
+          path: /var/tmp/build/ext-mongodb-${{ env.EXT_MONGODB_VERSION }}
+          key: ext-mongodb-${{ env.EXT_MONGODB_VERSION }}-php${{ matrix.php }}-${{ github.sha }}
```

Comment:
is `${{ github.sha }}` really needed here?
we only added it for composer to generate uniqueness so the cache is additive. doesn't' seem neccesary here

Reply:
We want to save the cache if anything has changed. This is the only way.